### DISCUSSION TITLE
MONSTERs

### DISCUSSION THREAD

**Discussion by adasue** [sonsuzus/BilsemPython#65] [Genel]
En iyi programcıları bekliyorum.
Ada Su - ÖYG Perşembe

![Monster](https://user-images.githubusercontent.com/114185391/209863923-74974bc6-d65d-495f-ad8f-181c2d78e78c.jpg)

**Comment by Akifatalar12** [comment]
`Merhaba! benim adım Muhammed Akif Atalar ben ÖYG-1 öğrencisiyim ve cumartesi grubundayım.Bu takım ilgimi çektiği için katılmak istiyorum.`

**Comment by ahmetsemihucar** [comment]
**_Merhaba, ben katılmak istiyorum !_**
- AD : **Ahmet Semih UÇAR**
- **ÖYG 1 , Cumartesi**

**Comment by EfeMertIrez** [comment]
Adım:Efe Mert İrez
Günüm:perşembe
Kademe:ÖYG2
Logonuz çok güzel

**Comment by TheDemr0** [comment]
Ad: Demir Akdemir
Öyg1
cumartesi
3 ve 4. dersler gidiyorum
Takımına katılmak istiyorum

**Comment by eyupbjk** [comment]
Ad : Eyüp Ensar
Öyg1 Cumartesi 1 ve 2. dersler 
Katılmak istiyorum

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Comment by Adaisland** [comment]
Merhaba,adım Ada Höbek
Ders saatlerim:Salı günü 5ve6
ÖYG2
Monsters takımına katılmak istiyorum

**Comment by adasue** [comment]
[EfeMertIrez](https://github.com/EfeMertIrez) 
[Akifatalar12](https://github.com/Akifatalar12) 
[ahmetsemihucar](https://github.com/ahmetsemihucar) 
[Adaisland](https://github.com/Adaisland) 
@duruozozer
Merhaba arkadaşlar grubuma hoş geldiniz.

**Reply by EfeMertIrez** [reply]
Whatsapp grubu falan açacak mısın

**Reply by ahmetsemihucar** [reply]
Bence açmalıyız hemde [1.Görev](https://github.com/sonsuzus/BilsemPython/discussions/81) için hazırlık yaparaız

**Comment by adasue** [comment]
Merhaba arkadaşlar whatsapp grubu kurdum rica etsem telefon numaranızı yazar mısınız iyi tatiller

**Reply by EfeMertIrez** [reply]
05416510810 - Efe Mert İrez

**Reply by ahmetsemihucar** [reply]
0544 772 16 76

**Comment by adasue** [comment]
[Akifatalar12](https://github.com/Akifatalar12)
[ahmetsemihucar](https://github.com/ahmetsemihucar)
arkadaşlar telefon numaralarınızı gönderiniz lütfen !!
